2022-06-02 21:22:37,565 INFO    MainThread:10217 [wandb_setup.py:_flush():76] Loading settings from /home/diego/.config/wandb/settings
2022-06-02 21:22:37,565 INFO    MainThread:10217 [wandb_setup.py:_flush():76] Loading settings from /home/diego/machine learning/machineLearnig_project1/Machine-learning-project-of-heart-with-MLops/source/notebooks/wandb/settings
2022-06-02 21:22:37,565 WARNING MainThread:10217 [wandb_setup.py:_flush():76] Unknown environment variable: WANDB_API
2022-06-02 21:22:37,565 INFO    MainThread:10217 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-06-02 21:22:37,565 INFO    MainThread:10217 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-06-02 21:22:37,565 INFO    MainThread:10217 [wandb_init.py:_log_setup():428] Logging user logs to /home/diego/machine learning/machineLearnig_project1/Machine-learning-project-of-heart-with-MLops/source/notebooks/wandb/run-20220602_212237-lj3wmqnp/logs/debug.log
2022-06-02 21:22:37,565 INFO    MainThread:10217 [wandb_init.py:_log_setup():429] Logging internal logs to /home/diego/machine learning/machineLearnig_project1/Machine-learning-project-of-heart-with-MLops/source/notebooks/wandb/run-20220602_212237-lj3wmqnp/logs/debug-internal.log
2022-06-02 21:22:37,565 INFO    MainThread:10217 [wandb_init.py:_jupyter_setup():378] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f52f6f73e50>
2022-06-02 21:22:37,565 INFO    MainThread:10217 [wandb_init.py:init():462] calling init triggers
2022-06-02 21:22:37,566 INFO    MainThread:10217 [wandb_init.py:init():465] wandb.init called with sweep_config: {}
config: {}
2022-06-02 21:22:37,566 INFO    MainThread:10217 [wandb_init.py:init():515] starting backend
2022-06-02 21:22:37,566 INFO    MainThread:10217 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-06-02 21:22:37,568 INFO    MainThread:10217 [backend.py:ensure_launched():217] starting backend process...
2022-06-02 21:22:37,570 INFO    MainThread:10217 [backend.py:ensure_launched():222] started backend process with pid: 10272
2022-06-02 21:22:37,570 INFO    MainThread:10217 [wandb_init.py:init():525] backend started and connected
2022-06-02 21:22:37,615 INFO    MainThread:10217 [wandb_run.py:_label_probe_notebook():1089] probe notebook
2022-06-02 21:22:37,616 INFO    MainThread:10217 [wandb_run.py:_label_probe_notebook():1099] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-06-02 21:22:37,616 INFO    MainThread:10217 [wandb_init.py:init():596] updated telemetry
2022-06-02 21:22:37,624 INFO    MainThread:10217 [wandb_init.py:init():627] communicating run to backend with 30 second timeout
2022-06-02 21:22:38,860 INFO    MainThread:10217 [wandb_run.py:_on_init():1924] communicating current version
2022-06-02 21:22:39,176 INFO    MainThread:10217 [wandb_run.py:_on_init():1928] got version response 
2022-06-02 21:22:39,177 INFO    MainThread:10217 [wandb_init.py:init():662] starting run threads in backend
2022-06-02 21:22:39,801 INFO    MainThread:10217 [wandb_run.py:_console_start():1898] atexit reg
2022-06-02 21:22:39,801 INFO    MainThread:10217 [wandb_run.py:_redirect():1771] redirect: SettingsConsole.WRAP
2022-06-02 21:22:39,802 INFO    MainThread:10217 [wandb_run.py:_redirect():1808] Wrapping output streams.
2022-06-02 21:22:39,803 INFO    MainThread:10217 [wandb_run.py:_redirect():1832] Redirects installed.
2022-06-02 21:22:39,804 INFO    MainThread:10217 [wandb_init.py:init():688] run started, returning control to user process
2022-06-02 21:22:41,913 INFO    MainThread:10217 [wandb_init.py:_pause_backend():341] pausing backend
2022-06-02 21:22:41,914 INFO    MainThread:10217 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-06-02 21:22:46,875 INFO    MainThread:10217 [wandb_init.py:_resume_backend():350] resuming backend
2022-06-02 21:22:46,879 INFO    MainThread:10217 [wandb_init.py:_pause_backend():341] pausing backend
2022-06-02 21:22:46,879 INFO    MainThread:10217 [jupyter.py:save_ipynb():377] not saving jupyter notebook
